axiom/client.go

```diff
@@ -352,9 +352,8 @@ func (c *Client) do(req *http.Request, v interface{}) (*response, error) {
 		return resp, err
 	}
 
-	key := limitKey(resp.Limit.limitType, resp.Limit.Scope)
 	c.limitsMu.Lock()
-	c.limits[key] = resp.Limit
+	c.limits[resp.Limit.limitType.String()] = resp.Limit
```

Comment:
doesn't have to be a map anymore does it?

Reply:
We still have different queries for ingest and query. `resp.Limit.limitType` is dynamic.

Reply:
Could be c.ingestLimit and c.queryLimit, no?

Reply:
Sure, but why deal with multiple fields?

Reply:
Removes the magic lookup string and is easier to grok. I won‘t die on this hill though, your decision